### ISSUE TITLE
[Bug]: Nextcloud Desktop 3.14.0 (MacOS): Translations are lost - user interface is shown in English

### ISSUE THREAD

**Issue by b-pfl** [nextcloud/desktop#7222]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Since the last update to Nextcloud 3.14.0, the user interface of the MacOS desktop client (universal) is shown in English and not in the local language as defined by the operating system. 
This is also subject to discussion at https://help.nextcloud.com/t/probleme-mit-update-auf-desktop-client-nextcloud-3-14-0-x86-64/204443
...

### Steps to reproduce

1. Update to 3.14.0
2. Start client, open random dialog
...

### Expected behavior

Interface texts are localized
...

### Which files are affected by this bug

Whole user interface

### Operating system

macOS

### Which version of the operating system you are running.

MacOS 14.7

### Package

Official macOS 12+ universal pkg

### Nextcloud Server version

29.0.7

### Nextcloud Desktop Client version

3.14.0

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 3.3.6 to 3.4.0)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by torstenhoppe** [comment]
Same problem here, even with 3.14.1
Working on Mac M1 with macOS Sequoia 15.0

**Comment by ywtoly** [comment]
Same problem here, even with 3.14.1
Working on Windows10 22h2(Simplified Chinese)

**Comment by raffisweb** [comment]
Same here with 3.14.3 on MacOS Sequoia. Interface English while system German.

**Comment by tomop-tg** [comment]
Same problem here with 3.15.2 on macOS Sequoia 15.2. The English interface is shown on my Japanese environment.

**Comment by e2ipi-1** [comment]
Same here with 3.15.3 on macOS Sequoia 15.2. Interface English while system French.

**Comment by benjamindahl** [comment]
Same here with 3.15.3 on macOS Sonoma 14.7.4

I can say, every team member in my office here has this exact same problem. Server version 29.0.12

**Comment by YouSly** [comment]
Same issue with 3.16.0 on multiple macos systems

Server Version is 31.0.0

**Comment by Rello** [comment]
Hallo,
yes, this is a known issue related to QT.
It is tracked in this enhancement
https://github.com/nextcloud/desktop/issues/111

It is on the roadmap but now confirmed release yet

**Comment by Rello** [comment]
> Issue [#111](https://github.com/nextcloud/desktop/issues/111) looks different for me.

yes, you are correct. but when we touch it anyway, we want to introduce this also because it covers a very common request.

> Cannot believe this is a framework issue.

We had to bump QT due to bugs with dependencies. We will check the next days...

**Comment by nilsding** [comment]
@rakekniven perfect :> 

for all the others: it will be part of the new daily builds for macOS (and 3.16.3 once it's released of course)

if this is an issue on Windows as well, please retry it with a new daily Windows build and open a new ticket containing the first few log lines (I'm interested in the ones containing `application.cpp:`.)
